### ISSUE TITLE
Windows might need 

### ISSUE THREAD

**Issue by maxnoe** [pep-dortmund/homepage-toolbox#135]
We had one system where localhost:0.0 was not working, and found that:
```
export DISPLAY=$(route.exe print | grep 0.0.0.0 | head -1 | awk '{print $4}'):0.0
```

works

**Comment by LuckyJosh** [comment]
Now I remember why the email from a few days ago rang a bell, see #118

**Comment by LuckyJosh** [comment]
Closed by #155
